Sources/TuistSupport/Utils/GitHandler.swift

```diff
@@ -63,11 +73,23 @@ public final class GitHandler: GitHandling {
         }
     }
 
+    public func lsRemote(url: String) throws -> String {
+        try capture(command: "git", "ls-remote", "-t", "--sort=v:refname", url)
+    }
+
     private func run(command: String...) throws {
         if Environment.shared.isVerbose {
             try system.runAndPrint(command, verbose: true, environment: System.shared.env)
         } else {
             try system.run(command)
         }
     }
+
+    private func capture(command: String...) throws -> String {
```

Comment:
why is this necessary? I think this should be done in `System.swift`, no?

Reply:
It's done this way to be consistent with the preceding `run` method. @luispadron, that was your creation, so do you have any thoughts on this?

Reply:
I don't recall why that's needed, it may not actually be required. But not a big deal otherwise.